spec/specification_spec.rb

```diff
@@ -291,6 +291,18 @@ module Pod
         @spec.subspec_by_name('Pod/Tests', false).should. nil?
       end
 
+      it "does return the test subspec given it's name when including test subspecs" do
+        @spec = Spec.new do |s|
+          s.name = 'Pod'
+          s.version = '1.0'
+          s.dependency 'AFNetworking'
+          s.osx.dependency 'MagicalRecord'
+          s.test_spec {}
+        end
+        test_spec = @spec.test_specs.first
+        @spec.subspec_by_name('Pod/Tests', false, true).should == test_spec
```

Comment:
the last parameter is set to `true` compared to the previous test which has it set to nothing (`false`)